### PR TITLE
Morae dev

### PR DESCRIPTION
**Pull request by Free-Quarks**
Updates the Rust Gromet struct to support v0.1.4 and preliminary role annotation for CHIME Gromets.  Adds in v0.1.4 Gromets to data directory.

### CODE REVIEW

File: data/gromet/examples/README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
-## GroMEt examples for testing
+## File Types in Example Directories
 
 The following files appear in the simple GroMet examples (not all have been generated for every example).
 
 The following are likely of most interest:
 
 - `*.py` : The source code file.
+- `*_gromet_FN_manual_diagram.pdf` : A manual drawing of the gromet wiring diagram
```
Comment (adarshp) [inline data/gromet/examples/README.md:8]: Can you update this file to remove mentions of files that are not actually in the repo?

File: skema/skema-rs/skema/src/bin/morae.rs [hunk 2]
```diff
@@ -3,20 +3,323 @@ use serde_json; // for json
...
+    // we now perform a check to make sure every entry in attributes has been captured and there are no duplicates
+    //let mut tot = 0; // This starting at 1 is because main is not counted as a dependency
+    for i in 0..subroutines_contracted.len() {
+        subroutines_contracted[i].indexes.sort();
+        subroutines_contracted[i].indexes.dedup();
+        //tot += subroutines_contracted[i].indexes.len();
+    }
+
+    // we now label each function, for coarse grain labeling
+    /*if tot == res.attributes.len() {
```
Comment (adarshp) [inline skema/skema-rs/skema/src/bin/morae.rs:225]: Can you remove this commented out code?

**Review by adarshp — CHANGES_REQUESTED** [review]
@Free-Quarks  - looks good, just a couple of minor changes (see my comments) and we should be good to merge the PR.

**Review by adarshp — APPROVED** [review]

File: data/gromet/examples/README.md (hunk 1 above)
Comment (Free-Quarks) [reply data/gromet/examples/README.md]: Yeah I updated this to remove a couple lines of files that don't exist anymore and added a couple lines of additional files I added to deal with some bugs in the gromets in v0.1.4 that I believe are now wrapped into the v0.1.5 update.

File: skema/skema-rs/skema/src/bin/morae.rs (hunk 2 above)
Comment (Free-Quarks) [reply skema/skema-rs/skema/src/bin/morae.rs]: Oh I missed that haha. Removed now!

**Pull request merged**